core/src/test/java/io/questdb/test/cairo/map/OrderedMapTest.java

```diff
@@ -537,7 +537,6 @@ public void testAsciiVarcharKey() throws Exception {
                     Utf8Sequence varchar = record.getVarcharA(1);
                     Assert.assertNotNull(varchar);
                     TestUtils.assertEquals(utf8Sink, varchar);
-                    Assert.assertFalse(varchar.isAscii());
```

Comment:
nit: would be nice to replace the removed lines with an update check.

Reply:
done in https://github.com/questdb/questdb/pull/4401/commits/765e27891d2394b5680b41a1483611d61cbb6cc6